internal/cmd/connect/command_custom_plugin_describe.go

```diff
@@ -0,0 +1,51 @@
+package connect
+
+import (
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/spf13/cobra"
+)
+
+func (c *customPluginCommand) newDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:         "describe <id>",
+		Short:       "Describe a custom connector plugin.",
+		Args:        cobra.ExactArgs(1),
+		RunE:        c.describe,
+		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin},
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Describe custom connector plugin",
+				Code: "confluent connect custom-plugin describe ccp-123456",
+			},
+		),
+	}
+
+	pcmd.AddContextFlag(cmd, c.CLICommand)
+	pcmd.AddOutputFlag(cmd)
+
+	return cmd
+}
+
+func (c *customPluginCommand) describe(cmd *cobra.Command, args []string) error {
+	plugin, err := c.V2Client.DescribeCustomPlugin(args[0])
+	if err != nil {
+		return err
+	}
+	out := &customPluginOut{
+		Id:                        plugin.GetId(),
+		Name:                      plugin.GetDisplayName(),
+		Description:               plugin.GetDescription(),
+		ConnectorClass:            plugin.GetConnectorClass(),
+		ConnectorType:             plugin.GetConnectorType(),
+		SensitiveConfigProperties: plugin.GetSensitiveConfigProperties(),
+	}
+	if output.GetFormat(cmd) == output.Human {
+		output.Println("Plugin Details")
```

Comment:
Our other commands avoid printing strings like this.
```suggestion
```

Reply:
I was actually following this https://github.com/confluentinc/cli/blob/main/internal/connect/command_cluster_describe.go#L98 

But I can remove it!